pkg/collector/corechecks/cluster/orchestrator/collectors/k8s/helpers.go

```diff
@@ -0,0 +1,121 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2025-present Datadog, Inc.
+
+//go:build kubeapiserver && orchestrator
```

Comment:
Fyi you only need to add the `test` build tag on files which are not tests themselves (but are only used for tests)
```suggestion
//go:build kubeapiserver && orchestrator && test
```